### ISSUE TITLE
Unexpected error when trying to view deleted tweet

### ISSUE THREAD

**Issue by cooljeanius** [kheina-com/Blue-Blocker#159]
Example link that causes an error: https://twitter.com/CathoIicConvert/status/1600301003424731136
It says "Sorry, that Tweet has been deleted." on the webpage, a BlueBlocker notification says "an error occurred! check the console and create an issue on [GitHub](https://github.com/kheina-com/Blue-Blocker/issues)", and the console says,
```
6:50:23.746 [Blue Blocker] unexpected error occurred while parsing request body 
Object { error: TypeError, body_str: '{"errors":[{"message":"_Missing: No status found with that ID.","locations":[{"line":5,"column":3}],"path":["threaded_conversation_with_injections_v2"],"extensions":{"name":"GenericError","source":"Server","code":144,"kind":"NonFatal","tracing":{"trace_id":"6e912c5dcae9e226"}},"code":144,"kind":"NonFatal","name":"GenericError","source":"Server","tracing":{"trace_id":"6e912c5dcae9e226"}}],"data":{}}', event: blue-blocker-event }
[chunk-67987f8a.js:1:16454](moz-extension://7969e46a-e5e2-48ac-8e73-92ba4bfad2ee/assets/chunk-67987f8a.js)
```
There are some little disclosure triangles that can be expanded to view more text from the error message in the console, but they go pretty deep... how much of that additional info do you need?

**Comment by DanielleMiu** [comment]
thanks! this should be plenty, I think this issue was fixed in v0.3.0. are you still using v0.2.7?

https://github.com/kheina-com/Blue-Blocker/blob/35b2de13d7edfc2c9668ac53eb10788ea65b7ad9/src/content/index.ts#L27-L32

**Comment by DanielleMiu** [comment]
oh, I see. not found responses for `TweetDetail` return a 200 response. very good coding there, twitter, very excellent.

**Comment by cooljeanius** [comment]
> thanks! this should be plenty, I think this issue was fixed in v0.3.0. are you still using v0.2.7?

just checked now and it says 0.3.1

**Comment by DanielleMiu** [comment]
this should be fixed in version `v0.3.2`
https://github.com/kheina-com/Blue-Blocker/blob/71e5e7316bedccc4db50bf2a49b58e9ebbef55ec/src/content/index.ts#L39-L43
